spec/system/admin/creating_an_estimate_spec.rb

```diff
@@ -2,9 +2,10 @@
 
 require "rails_helper"
 
-RSpec.describe "Creating an estimate" do
```

Comment:
is the :js param necessary here?

Reply:
Yes. Without it the JS doesn't load and the changes in this PR use the JS code